src/components/lyne-panel/lyne-panel.scss

```diff
@@ -0,0 +1,93 @@
+:host {
```

Comment:
We did not really talk about yet how far we want to go with RTL support. In this case, we could start changing the alignment of the text and the position of the triangles.

Reply:
Ok, did it. I only had to change border-bottom, border-top and border-right to according logical properties